_sass/_layout.scss

```diff
@@ -52,7 +52,7 @@ h1 {
 
 }
 .front-image {
-  max-height: 20em;
+  max-height: 18em;
```

Comment:
This will modify all images. I recommend not to push it through this pull request. Remove before merging.